versions.props

```diff
@@ -2,50 +2,50 @@ com.boundary:high-scale-lib = 1.0.6
 com.ea.agentloader:ea-agent-loader = 1.0.3
 com.fasterxml.jackson.*:* = 2.15.2
 com.fasterxml.jackson.core:jackson-databind = 2.15.2
-com.github.ben-manes.caffeine:caffeine = 3.1.5
+com.github.ben-manes.caffeine:caffeine = 3.1.8
 com.github.peterwippermann.junit4:parameterized-suite = 1.1.0
 com.github.rholder:guava-retrying = 2.0.0
 com.github.stefanbirkner:system-rules = 1.19.0
 com.github.tomakehurst:wiremock-standalone = 2.27.2
-com.google.auto.service:auto-service = 1.0.1
+com.google.auto.service:auto-service = 1.1.1
```

Comment:
This bump fails with `@AutoService cannot be applied to an abstract class or an interface` due to https://github.com/google/auto/commit/591731c32a6b425a43023335fb062c247fd65e63 so we may need to pin on an older version or update our uses of `@AutoService` with `@Value.Immutable` types, for example:

```
@AutoService(KeyValueServiceRuntimeConfig.class)
@JsonSerialize(as = ImmutableDbKeyValueServiceRuntimeConfig.class)
@JsonDeserialize(as = ImmutableDbKeyValueServiceRuntimeConfig.class)
@JsonTypeName(DbAtlasDbFactory.TYPE)
@Value.Immutable
public abstract class DbKeyValueServiceRuntimeConfig implements KeyValueServiceRuntimeConfig {
```